packages/amplify_api_rest/lib/method_channel_api_rest.dart

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'dart:collection';
+import 'dart:typed_data';
+import 'package:amplify_api_rest_plugin_interface/amplify_api_rest_plugin_interface.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+import 'dart:convert';
+
+import 'amplify_api_rest.dart';
+
+const MethodChannel _channel =
+    MethodChannel('com.amazonaws.amplify/amplify_api_rest');
+
+/// An implementation of [AmplifyPlatform] that uses method channels.
+class AmplifyAPIRestMethodChannel extends AmplifyAPIRest {
+  RestOperation _restFunctionHelper(
+      {@required String methodName, @required RestOptions restOptions}) {
+    try {
+      // Send Request Code to Native
+      String code = UUID.getUUID();
+
+      Future<RestResponse> futureResponse =
+          _callNativeRestMethod(methodName, code, restOptions);
```

Comment:
we should call `code` either `cancelCode` or `cancelToken` or something similar to specify it's a code for canceling,

Reply:
good point.  Updated all code -> cancelToken across all platforms in our merged branch